cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -622,6 +624,20 @@ func (s *Server) handleReIndex(w http.ResponseWriter, r *http.Request) {
 	_ = repoTmpl.Execute(w, data)
 }
 
+func handleHealthCheck(w http.ResponseWriter, r *http.Request) {
+	switch r.Header.Get("Accept") {
+	case "application/json":
+		w.Header().Set("content", "application/json")
+		payload, err := json.Marshal(hc)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+		_, _ = w.Write(payload)
```

Comment:
If we're throwing away both results, couldn't we just not assign it to anything?
```suggestion
		w.Write(payload)
```

Reply:
I admit I am heavily influenced by Goland complaining about unhandled errors ;-) I scanned the standard library a bit and they seem to follow you suggestion so let's do that.